Description: Fix missing libncurses5 dependency
Author: Sam Hocevar <sho@debian.org>
Reviewed-by: Herbert Parentes Fortes Neto <hpfn@ig.com.br>
Last-Update: 2015-08-13
Index: libcdk5-5.0.20141106/Makefile.in
===================================================================
--- libcdk5-5.0.20141106.orig/Makefile.in
+++ libcdk5-5.0.20141106/Makefile.in
@@ -290,7 +290,7 @@ $(OBJECTS) :	include/cdk_config.h
 all cdklib :: $(CDKLIB)
 
 @MAKE_NORMAL@cdkshlib $(OLD_SHLIB_FILE) :: $(CDKSHOBJS)
-@MAKE_NORMAL@	gcc -shared -Wl,-soname=$(OLD_SONAME) $(LDFLAGS) $(LIBS) -o $(OLD_SHLIB_FILE) $(CDKSHOBJS)
+@MAKE_NORMAL@	gcc -shared -Wl,-soname=$(OLD_SONAME) $(LDFLAGS) $(LIBS) -o $(OLD_SHLIB_FILE) $(CDKSHOBJS) $(LIBS)
 
 #
 # Make the examples directory.
